rustls/src/server/server_conn.rs

```diff
@@ -485,11 +485,12 @@ impl Acceptor {
         Ok(Self::default())
     }
 
-    /// Returns true if the caller should call [`Connection::read_tls()`] as soon as possible.
-    ///
-    /// For more details, refer to [`CommonState::wants_read()`].
+    /// By analogy with [`CommonState::wants_read()`], this returns true to indicate that
```

Comment:
Nit: would still like to keep the single self-contained line that summarizes the method.